mne/io/base.py

```diff
@@ -1596,7 +1596,6 @@ def crop(self, tmin=0.0, tmax=None):
 
         if self.annotations is not None:
             annotations = self.annotations
-            annotations.onset -= tmin
```

Comment:
since annotations times are all in seconds wrt to absolute time this is no need subtract anything when cropping.